php/7.4/overrides.conf

```diff
@@ -0,0 +1,4 @@
+[www]
+; a custom user that matches the host's user ID and created in the Dockerfile
+user = squareone
+group = squareone
```

Comment:
I have been using this log format below and has been super helpful when debugging. It gives you how much time, cpu, memory, url for each request.  
```
access.format = "[%t] %m %{REQUEST_SCHEME}e://%{HTTP_HOST}e%{REQUEST_URI}e %f pid:%p took:%ds mem:%{mega}Mmb cpu:%C%% status:%s {%{REMOTE_ADDR}e|%{HTTP_USER_AGENT}e}"
```